### ISSUE TITLE
Improve Filter button accessibility through more fitting ARIA attribute & focus management

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#625]
## Description

Using a screen reader, when Filter button is activated, it says "Current". This doesn't appear to fit the Filter button's behavior.

When the filter button is selected, we display the filter options. It is acting like a disclosure. In this case, the filter should have an expanded and collapsed state. When it is expanded, the next element where the screen reader focus should fall on is the heading "Suggested Priority", and so on.

<details>
<summary>Screenshot</summary>

<img width="626" alt="Screenshot 2024-05-05 at 16 55 14" src="https://github.com/CodeForPhilly/clean-and-green-philly/assets/102739972/c16a1e0f-2fbb-4df4-9b33-6b939fbb6a06">

</details>

## Acceptance Criteria

- [ ] use `<aria-expanded>` on Filter button
- [ ] When the Filter is expanded, the next focus (screenreader) should be the items in the Filter options
- [ ] When the Filter option is dismissed, focus stays on the Filter button.

**Comment by Amberroseweeks** [comment]
Hello, i'd like to be assigned this task if it is available, thank you!

**Comment by bacitracin** [comment]
Assigned!
